plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemColor.java

```diff
@@ -79,6 +81,13 @@ public ObjectTag getObjectAttribute(Attribute attribute) {
                 }
                 return BukkitColorExtensions.fromColor(pm.getColor()).getObjectAttribute(attribute.fulfill((1)));
             }
+            if (mat == Material.FILLED_MAP) {
+                MapMeta mapMeta = (MapMeta) item.getItemMeta();
+                if (!mapMeta.hasColor()) {
+                    return BukkitColorExtensions.fromColor(Color.WHITE).getObjectAttribute(attribute.fulfill(1));
```

Comment:
Is returning `WHITE` here correct? Seems like it should probably be `null`?

Reply:
tbh... i dont know which default color of filled_map 😅, that why i put white color. i'll change it asap, thanks for remind me to using null